roles/ansible_service_broker/defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+ansible_service_broker_image: "ansibleplaybookbundle/ansible-service-broker:latest"
+ansible_service_broker_etcd_image: "quay.io/coreos/etcd:latest"
+ansible_service_broker_state: "{{ 'absent' if ansible_service_broker_remove|default(false) else 'present' }}"
```

Comment:
can we do a 'install' and 'remove' logic within the role instead of controlling state this way?